be/src/storage/storage_engine.cpp

```diff
@@ -109,6 +109,23 @@ void StorageEngine::load_data_dirs(const std::vector<DataDir*>& data_dirs) {
     threads.reserve(data_dirs.size());
     for (auto data_dir : data_dirs) {
         threads.emplace_back([data_dir] {
+            uint64_t live_sst_files_size_before = 0;
+            if (!data_dir->get_meta()->get_live_sst_files_size(&live_sst_files_size_before)) {
+                LOG(WARNING) << "data dir " << data_dir->path() << " get_live_sst_files_size failed";
+            }
+            Status s = data_dir->get_meta()->compact();
```

Comment:
Now we test the RocksDB stall in seeking.
How about the metadata is huge for many tablets and loads, does the compact() will consume much time?
Even the compact is not necessary.
I think it's better to test the compact to huge meta for 10GB or even more.

Reply:
In another problem, I had test the large meta, the size is 973G, scale according ratio, the compact will not take too much time.

Reply:
It will not take too much time, 20G cost 28 secs.
```
** Compaction Stats [meta] **
Level    Files   Size     Score Read(GB)  Rn(GB) Rnp1(GB) Write(GB) Wnew(GB) Moved(GB) W-Amp Rd(MB/s) Wr(MB/s) Comp(sec) CompMergeCPU(sec) Comp(cnt) Avg(sec) KeyIn KeyDrop Rblob(GB) Wblob(GB)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  L0      0/0    0.00 KB   0.0      0.0     0.0      0.0       0.0      0.0       0.0   1.0      0.0     30.8      0.05              0.00         1    0.053       0      0       0.0       0.0
  L1      0/0    0.00 KB   0.0      0.1     0.0      0.1       0.0     -0.1       0.0   1.6    597.7     23.6      0.11              0.10         1    0.110     36K    27K       0.0       0.0
  L2      0/0    0.00 KB   0.0      2.4     0.0      2.4       0.0     -2.4       0.0   3.6    901.0      3.4      2.72              2.57         1    2.717     77K    76K       0.0       0.0
  L3      1/0    8.85 MB   0.0     17.1     0.0     17.1       0.4    -16.7       0.0  44.8    688.5     16.2     25.50             19.87        11    2.318    758K   740K       0.0       0.0
 Sum      1/0    8.85 MB   0.0     19.6     0.0     19.6       0.4    -19.2       0.0 259.2    707.2     15.0     28.38             22.55        14    2.027    872K   844K       0.0       0.0
 Int      0/0    0.00 KB   0.0     19.6     0.0     19.6       0.4    -19.2       0.0 445371721.0    708.5     15.0     28.33             22.55        13    2.179    872K   844K       0.0       0.0

** Compaction Stats [meta] **
Priority    Files   Size     Score Read(GB)  Rn(GB) Rnp1(GB) Write(GB) Wnew(GB) Moved(GB) W-Amp Rd(MB/s) Wr(MB/s) Comp(sec) CompMergeCPU(sec) Comp(cnt) Avg(sec) KeyIn KeyDrop Rblob(GB) Wblob(GB)
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 Low      0/0    0.00 KB   0.0     19.6     0.0     19.6       0.4    -19.2       0.0   0.0    708.5     15.0     28.33             22.55        13    2.179    872K   844K       0.0       0.0
User      0/0    0.00 KB   0.0      0.0     0.0      0.0       0.0      0.0       0.0   0.0      0.0     30.8      0.05              0.00         1    0.053       0      0       0.0       0.0

Blob file count: 0, total size: 0.0 GB

Uptime(secs): 31.2 total, 31.1 interval
Flush(GB): cumulative 0.002, interval 0.000
AddFile(GB): cumulative 0.000, interval 0.000
AddFile(Total Files): cumulative 0, interval 0
AddFile(L0 Files): cumulative 0, interval 0
AddFile(Keys): cumulative 0, interval 0
Cumulative compaction: 0.42 GB write, 13.67 MB/s write, 19.60 GB read, 643.36 MB/s read, 28.4 seconds
Interval compaction: 0.41 GB write, 13.66 MB/s write, 19.60 GB read, 645.45 MB/s read, 28.3 seconds
Stalls(count): 0 level0_slowdown, 0 level0_slowdown_with_compaction, 0 level0_numfiles, 0 level0_numfiles_with_compaction, 0 stop for pending_compaction_bytes, 0 slowdown for pending_compaction_bytes, 0 memtable_compaction, 0 memtable_slowdown, interval 0 total count
Block cache LRUCache@0x8d1fc30 capacity: 8.00 MB collections: 1 last_copies: 1 last_secs: 1.4e-05 secs_since: 31
Block cache entry stats(count,size,portion): OtherBlock(43,1.63 MB,20.3419%) Misc(2,4.58 KB,0.0558853%)

** File Read Latency Histogram By Level [meta] **

** DB Stats **
Uptime(secs): 31.2 total, 31.2 interval
Cumulative writes: 0 writes, 0 keys, 0 commit groups, 0.0 writes per commit group, ingest: 0.00 GB, 0.00 MB/s
Cumulative WAL: 0 writes, 0 syncs, 0.00 writes per sync, written: 0.00 GB, 0.00 MB/s
Cumulative stall: 00:00:0.000 H:M:S, 0.0 percent
Interval writes: 0 writes, 0 keys, 0 commit groups, 0.0 writes per commit group, ingest: 0.00 MB, 0.00 MB/s
Interval WAL: 0 writes, 0 syncs, 0.00 writes per sync, written: 0.00 GB, 0.00 MB/s
Interval stall: 00:00:0.000 H:M:S, 0.0 percent
```